ambari-server/src/main/resources/stacks/BIGTOP/3.2.0/services/YARN/quicklinks/quicklinks.json

```diff
@@ -20,7 +20,7 @@
         "label": "ResourceManager UI",
         "requires_user_name": "false",
         "component_name": "RESOURCEMANAGER",
-        "url": "%@://%@:%@",
+        "url": "%@://%@:%@/ui2",
```

Comment:
This is required going forward?

Reply:
Yes, in Ambari, the default Yarn UI redirect link will be "url": "%@://%@:%@/ui2",
However, the old UI can still be accessed through "url": "%@://%@:%@"